src/framework/components/element/component.js

```diff
@@ -43,10 +84,51 @@ pc.extend(pc, function () {
             _warning = true;
         }
     };
+    
     ElementComponent = pc.inherits(ElementComponent, pc.Component);
 
-
     pc.extend(ElementComponent.prototype, {
+        
+        // Prepares stencil params for the inner components to be utilized during
+        // rendering. To keep things the least obtrusive way, it assumes the default stencil
+        // buffer value is 0, meaning the topmost mask (mind masking elements can be nested 
+        // into each other) should fill the buffer with a const using GREATEREQUAL function,
+        // while children should be drawn with smaller ref value and LESSEQUAL function.
+        _getStencilParameters: function() {
+            return new pc.StencilParameters({
```

Comment:
Is masking changed once in a lifetime of an application for only one element? If it is called few times, and for each element individually, then this should be pre-allocated, as it will produce GC stalls.